bodhi/server/models.py

```diff
@@ -2197,6 +2253,9 @@ def meets_testing_requirements(self):
         """
         num_days = self.mandatory_days_in_testing
 
+        if config.get('ci.required', False) and not self.ci_passed:
```

Comment:
One thing to note that's pretty important: Bodhi's settings will always be strings - so if ```ci.required``` is "False", this will actually interpret it to be True since ```bool("False") is True```.

This config is used in lots of places in this PR, so be sure to fix the other uses of it too.

This has been the subject of other Bodhi bugs in the past. I really want to build a central settings interpreter that validates all settings, applies all defaults (so you don't have to remember to type ```, False``` everywhere like here), and converts all settings to their logical types (like bools, or lists, or whatever). It's a real pain how it is, and it's error prone. I did file a ticket about this:

https://github.com/fedora-infra/bodhi/issues/1489